### PR TITLE
review on 4/9/22

### PR DESCRIPTION
**Pull request by rockerinnight**

### CODE REVIEW

**Comment by DienMinh** [comment]
@rockerinnight

File: src/app/app.component.ts [hunk 1]
```diff
@@ -57,34 +57,29 @@ export class AppComponent {
...
-          }
-          if (x > y) {
-            return 1;
-          }
-          return 0;
-        }
-      });
+      this.usersSort = [...this.users];
     }
+    this.usersSort.sort((a, b) => {
```
Comment (rockerinnight) [inline src/app/app.component.ts:65]: missing return here

**Review by rockerinnight — COMMENTED** [review]
@DienMinh

**Comment by rockerinnight** [comment]
good. merged.

**Pull request merged**